inc/hclib-task.h

```diff
@@ -32,32 +32,20 @@ typedef struct _hclib_task_t {
      * and locality flexible asyncAny
      */
     int is_async_any_type;
```

Comment:
I forgot that GitHub sticks a stupid Merge Commit in the history after every PR is merged. Without that, the diffs would have shrunk on their own as we worked through them in order... Due to the merge commits, I'll have to rebase each successive PR as we move along, but that's still not too bad. Like I said, I thought PR comment made it very clear that these changes are cumulative, and so you'd have no motivation to look at them out of order. I guess I should have prefixed the PR names with something like "[PART 1]", "[PART 2]" etc. to make this more obvious.... But you guys didn't have any problem getting through the first 3 PRs (and it seemed like you both understood that *those* were cumulative), so I didn't have any reason to think that you guys would stall on the remainder of the chain.